R/check-devtools.r

```diff
@@ -21,7 +21,7 @@ check_dev_versions <- function(pkg = ".") {
   message("Checking for dependencies on development versions... ",
     appendLF = FALSE)
 
-  deps <- pkg_deps(pkg, NA)
+  deps <- dev_package_deps(pkg, NA)
   deps <- deps[!is.na(deps$version), , drop = FALSE]
```

Comment:
Right, and also the line below. The two functions are not interchangeable. I'm closing this pull.